dev-qt/qtcore/qtcore-5.14.9999.ebuild

```diff
@@ -3,15 +3,15 @@
 
 EAPI=7
 QT5_MODULE="qtbase"
-inherit qt5-build
+inherit linux-info qt5-build
 
 DESCRIPTION="Cross-platform application development framework"
 
 if [[ ${QT5_BUILD_TYPE} == release ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
 fi
 
-IUSE="icu systemd"
+IUSE="icu old-kernel systemd"
```

Comment:
do we have to add `kernel_linux` here? I forgot how this works... 🙈

Reply:
It is already provided by `linux-info.eclass`